composer.json

```diff
@@ -15,7 +15,7 @@
         }
     },
     "require": {
-        "php": ">=7.1",
+        "php": ">=7.2",
```

Comment:
What do you think about using the `^` operator, though?

At this time we can't be sure yet this library will work with PHP 9000.

Reply:
Good point.